src/utils/getCurrentOnChainDistribution.ts

```diff
@@ -22,6 +22,21 @@ export interface Proofs {
       | undefined;
   };
 }
+export const getAllProofs = () => {
+  const proofs: Proofs[] = [];
+  for (let index = numberOfEpochs; index > 0; index--) {
+    const filename = `proofs-${index}.json`;
+    let lastProofRaw: any;
+    try {
+      lastProofRaw = require(`../../distribution/proofs/${filename}`);
+    } catch (e: any) {
+      if (e.code !== "MODULE_NOT_FOUND") throw Error(e);
```

Comment:
What if one of the proof can't be found? Shouldn't we at leat display a warning?

Reply:
If proof is not found is just because proof does not exist. This is not a warning but a known behavior I think. The problem is more about the method, which is not really clean